<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>723</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>Welcome to Jeremy's En525.743 Embedded Development Receiver Client</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Enable_receiver">
        <property name="text">
         <string>Receive from BeagleBone</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="configlayout">
        <item row="7" column="1">
         <widget class="QLabel" name="mp3_location_label">
          <property name="text">
           <string>mp3 name/location</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="modulationtype_label">
          <property name="text">
           <string>Modulation Type</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="mp3_location">
          <property name="text">
           <string>/home/jeremy/en525_743/recording.mp3</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="enable_speakers">
          <property name="text">
           <string>Speakers</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="port_label">
          <property name="text">
           <string>BeagleBone Control Port</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="beaglebone_port">
          <property name="text">
           <string>1234</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QSpinBox" name="CB_channel_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QComboBox" name="modulation_combobox">
          <item>
           <property name="text">
            <string>FM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CB Radio - AM</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="fm_freq_BOX">
          <property name="text">
           <string>97900000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="ip_label">
          <property name="text">
           <string>BealeBone IP</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="beaglebone_ip">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="fm_freq_label">
          <property name="text">
           <string>FM - Frequency</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="cb_channel_label">
          <property name="text">
           <string>CB - Channel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QCheckBox" name="enable_recording">
          <property name="text">
           <string>Record to mp3 file</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="output_label">
          <property name="text">
           <string>Output</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="data_port_label">
          <property name="text">
           <string>BeagleBone Data Port</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="beaglebone_data_port">
          <property name="text">
           <string>5678</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
